### ISSUE TITLE
Fix training loops and sentence transformer

### ISSUE THREAD

**Issue by thefirebanks** [wri-dssg/policy-data-analyzer#72]
- [x] Evaluation code should be refactored to only take care of calculating results, and storing of results should be done in a different area
- [x] Evaluation should be done on validation set, not test set 
- [x] Add method to evaluate on test set

**Comment by thefirebanks** [comment]
Fixed by #81
